tests/datasets/test_sessions_processing.py

```diff
@@ -85,6 +84,12 @@ def query_runner(
             "selected_columns": ["sessions", "bucketed_started"],
             "groupby": ["bucketed_started"],
             "granularity": 60,
+            # XXX: using `conditions` here, as `from_date`/`to_date` is not being
```

Comment:
from_date and to_date are parsed by query extensions. See this function: https://github.com/getsentry/snuba/blob/master/snuba/request/schema.py#L208

Your test below is only running the processors.